content/question/1.md

```diff
@@ -0,0 +1,103 @@
+---
+Answers:
+- Accepted: true
+  BodyMarkdown: "Where in you'r code have you applied the force ?\r\nare you absolutely\
+    \ sure it is not in the onUpdate() or similar never ending call ?\r\n\r\nWhat\
+    \ is the purpose?\r\n\r\nhave you got friction in the joint ?\r\naccording to\
+    \ newtons laws of motion, the arm will continue to rotate forever, unless something\
+    \ provides a reaction force, like friction.\r\n\r\nif you apply the force in a\
+    \ small duration of time, the turnrate of the arm will after that moment in time\
+    \ become constant. With frictions it will decellerate, with repeated force the\
+    \ arm will accellerate.\r\nwith repeated force and friction the arm will reach\
+    \ a steady state where you'r applied force equals the force from friction, and\
+    \ from here the speed will eventually become constant.\r\n\r\n**solutionset:**\r\
+    \n\r\n1. if you are repeatingly applying force, stop that after some time and\
+    \ let the friction stop the arm (use gazebos simulation time..)\r\n\r\n2. learn\
+    \ control theory, (and basic physics) and let a controller that calculate the\
+    \ desired force to have the arm move to a position or speed or both be implemented.\
+    \  (You should do this to control the HW anyway)\r\n\r\n3. stop using forces,\
+    \ _cheat_ and use the animation interface for gazebo\r\n"
```

Comment:
This format parses but it isn't the literal scalar format we discussed. Was it not possible to generate scalar literals after all?

Reply:
I'm having a weird problem there. Some questions are generated as scalar literals and some are not. I'm still debugging why is this happening

Reply:
Just to save every step I take from here:

If the dumped string contains a "\r" character, the string is dumped in quotes (that's my investigation for now, I'm not sure why is that.

Reply:
Now it seems everything has the literal now. (Just replaced "\r" for "" in all body markdowns)